incident-response-plan.md

```diff
@@ -0,0 +1,153 @@
+# Incident Response Plan
+
+## Purpose  
+This document outlines the process for handling security-impacting incidents affecting one or more projects in the Foundation’s ecosystem. Incidents may include platform changes with security implications, account compromise, or other security events that require coordinated response.
+
+The Foundation’s role is to:  
+1. **Receive and triage THE REPORTS**  
+2. **Connect reporters and affected maintainers with the right experts**  
+3. **Facilitate coordinated response** across multiple projects when needed  
+4. **Communicate clearly and act as the contact point** while respecting confidentiality and responsible disclosure principles  
+
+---
+
+## Scope  
+
+This plan covers incidents such as:  
+- **Platform changes or providers outages** (e.g., GitHub UI change) that create security or operational risk.  
```

Comment:
I do not think we should use a Github UI change in this doc. Folks changing around UI elements while not ideal is not something that should be covered by a security incident doc. 

However something like the following would belong:

Platform changes or provider outages with security implications (e.g., compromised authentication systems, unexpected data exposure, service disruptions affecting security controls).